Source/JavaScriptCore/API/JSValueRef.cpp

```diff
@@ -390,12 +390,11 @@ JSValueRef JSValueMakeFromJSONString(JSContextRef ctx, JSStringRef string)
     JSGlobalObject* globalObject = toJS(ctx);
     JSLockHolder locker(globalObject);
     String str = string->string();
-    unsigned length = str.length();
-    if (!length || str.is8Bit()) {
-        LiteralParser<LChar> parser(globalObject, str.characters8(), length, StrictJSON);
+    if (str.isEmpty() || str.is8Bit()) {
+        LiteralParser<LChar> parser(globalObject, str.span8(), StrictJSON);
```

Comment:
Can `LiteralParser` deduce the template argument?

Reply:
Yes it can. I'll drop the template arguments.